app/assets/scripts/components/dashboard/sources-card.js

```diff
@@ -29,13 +29,36 @@ export default function SourceInfo({ sources }) {
           return (
             <SourceList>
               {sources.map(source => (
-                <Source
-                  key={source.name}
-                  to={source.sourceURL || source.url}
-                  title={source.name}
-                >
-                  {source.name}
-                </Source>
+                <li key={source.id}>
+                  {source.url ? (
+                    <a
+                      className="source__title source__title--external"
+                      href={source.url}
+                      rel="noreferrer noopener"
+                      target="_blank"
+                    >
+                      {source.name}
+                    </a>
+                  ) : (
+                    <p className="source__title">{source.name}</p>
+                  )}
+                  <p className="source__readme">
+                    {source.readme ? (
+                      <a
+                        href={`${config.api}/${source.readme.replace(
+                          /^\/v2\//,
+                          ''
+                        )}`}
+                        rel="noreferrer noopener"
+                        target="_blank"
+                      >
+                        Technical readme
+                      </a>
+                    ) : (
+                      'No technical readme for this source'
```

Comment:
If there is no technical readme, should we just skip it instead?

Reply:
This was specifically requested in the issue.

Reply:
Yea, I guess I could go either way.  I added that to the issue because I kept forgetting that only a small number of datasets have technical readmes.  I kept getting frustrated looking for them and wanted a lack of readme to be specified.  Users may not be actively looking for them, though, so perhaps the text is unnecessary.  Keep the "no readme" text or not @danielfdsilva you can make the final call.